### DISCUSSION TITLE
Adguard during installation

### DISCUSSION THREAD

**Discussion by semaf** [RaspAP/raspap-webgui#1188] [Ideas]
Hey,

Would be nice if during installation of RaspAP the installation of Adguard is requested. 

If possible even Pihole as an option.

**Comment by billz** [comment]
We generally don't install third-party applications, beyond what is required for RaspAP to work. Our FAQ has a dedicated "integrations" section that covers many popular apps, including Adguard.

Re: Pi-Hole, see https://docs.raspap.com/faq/#pihole
